### PR TITLE
Setting to enable/disable email notifications for followed resources

### PR DESCRIPTION
**Pull request by 2e2a**

### CODE REVIEW

File: src/adhocracy_core/docs/notification.rst [hunk 1]
```diff
@@ -43,8 +43,8 @@ Following
 On default you don`t follow any resource:
 
     >>> resp = app_participant.get(participant_path).json
-    >>> resp['data'][INotification.__identifier__]
-    {'follow_resources': []}
+    >>> resp['data'][INotification.__identifier__].get('follow_resources')
+    []
```
Comment (der-john) [inline src/adhocracy_core/docs/notification.rst]: Is this related? If so, how?

File: src/adhocracy_core/adhocracy_core/sheets/notification.py [hunk 2]
```diff
@@ -37,6 +39,7 @@ class NotificationSchema(MappingSchema):
 
     follow_resources = UniqueReferences(reftype=NotificationFollowReference,
                                         choices_getter=get_follow_choices)
+    email_notification_enabled = Boolean(default=True, missing=drop)
```
Comment (2e2a) [inline src/adhocracy_core/adhocracy_core/sheets/notification.py]: missing=drop means that if the value is None its not set at all. This allows setting just the required fields of a sheet.

File: src/adhocracy_core/docs/notification.rst (hunk 1 above)
Comment (2e2a) [inline src/adhocracy_core/docs/notification.rst]: As data now has "follow_resources" and "email_notification_enable" this emphasizes the point above "On default you don`t follow any resource".

Comment (der-john) [inline src/adhocracy_core/docs/notification.rst]: aaaah now I get it.
